backend/src/models/userModel.ts

```diff
@@ -140,16 +140,12 @@ export default class UserModel {
             await transaction.rollback();
             throw error;
         }
-    }
-
```

Comment:
was there a reason this function was removed? I am running into a build error when I start the backend cause authController calls this